Remove unused dependency - opencollective ?

Are you using the command line tool `less-watch-compiler`?
Yes

If you are using the command line tool, which version are you using (`less-watch-compiler --version` to find out)?
1.14.1

Is the issue reproducible after updating to the latest version (` npm update less-watch-compiler`)?
Yes

Steps to reproduce:

-  ~~`npm install`  on the project directory~~
-  `rm -rf ./node_modules/ && npm install`  on the project directory

Actual Results:

- `npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.`

Expected Results:

-  No deprecation warnings

Here is the `npm ls core-js` result. Also, figured out that https://github.com/jonycheung/deadsimple-less-watch-compiler/pull/79 added `opencollective` dependency which is really not used anywhere or needed.
```
$ npm ls core-js
+-- less-watch-compiler@1.14.1
| `-- opencollective@1.0.3
|   `-- babel-polyfill@6.23.0
|     +-- babel-runtime@6.26.0
|     | `-- core-js@2.6.11
|     `-- core-js@2.6.11
```

Same error here:
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! less-watch-compiler@1.14.4 postinstall: `opencollective-postinstall`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the less-watch-compiler@1.14.4 postinstall script.

--- a/package.json
+++ b/package.json
@@ -33,11 +33,11 @@
     "global": "^4.3.1",
     "less": "^3.8.1",
     "opencollective-postinstall": "^2.0.1",
-    "shelljs": ">= 0.4.0",
-    "opencollective": "^1.0.3"
+    "shelljs": ">= 0.4.0"
   },
   "engine": "node 0.10.26",
   "devDependencies": {
+    "opencollective": "^1.0.3",
     "babel-cli": "6.26.0",
     "babel-preset-env": "1.7.0",
     "cz-conventional-changelog": "2.1.0",

